src/tasks/addonindex/taskaddonindex.cpp

```diff
@@ -34,37 +35,47 @@ void TaskAddonIndex::run() {
     connect(request, &NetworkRequest::requestCompleted, this,
             [this](const QByteArray& data) {
               logger.debug() << "Get addon index completed";
+
               m_indexData = data;
               maybeComplete();
             });
   }
 
-  // Index file signature
-  {
-    NetworkRequest* request = NetworkRequest::createForGetUrl(
-        this, QString("%1manifest.json.sign").arg(Constants::addonSourceUrl()),
-        200);
+  if (Feature::get(Feature::Feature_addonSignature)->isSupported()) {
+    // Index file signature
+    {
```

Comment:
we can remove this '{' and the corresponding '}'. It's already in the if-stmt block.